AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/SecondTabViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  SecondTabViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Eashir Arafat on 10/6/16.
+//  Copyright © 2016 Evan. All rights reserved.
+//
+
+import UIKit
+
+class SecondTabViewController: UIViewController, UITextFieldDelegate {
+    
+    var universalValue: Int?
+    var setValue: Int?
+    
+    @IBOutlet weak var numLabel: UILabel!
+    @IBOutlet weak var textField: UITextField!
+    @IBOutlet weak var stepper: UIStepper!
+    @IBOutlet weak var slider: UISlider!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func sliderValues(_ sender: UISlider) {
+        if universalValue == nil {
+            universalValue = Int(sender.value)
+            numLabel.text = String(describing: universalValue!)
+        }
+        else {
+        universalValue = setValue
+        numLabel.text = String(describing: universalValue)
+        }
+        
+    }
+  
+    @IBAction func stepValues(_ sender: UIStepper) {
+        
+        if universalValue == nil {
```

Comment:
You can use a helper function like 
`
func updateLabelValue(_ value: Int) {
        currentValueLabel.text = String(value)
        slider.value = Float(value)
        stepper.value = Double(value)
        textField.text = String(value)
    }
`